purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/properties/ColorInfo.kt

```diff
@@ -57,6 +57,9 @@ sealed interface ColorInfo {
             @Serializable(with = RgbaStringArgbColorIntDeserializer::class)
             @ColorInt
             @get:JvmSynthetic val color: Int,
+            /**
+             * A percentage value in the range 0.0..100.0.
```

Comment:
Nitpick should this be ints intead of floats?
```suggestion
             * A percentage value in the range 0..100.
```

Reply:
Oh yea the underlying field is a `Float` so that's why, but I think for documentation purposes an int is clearer, especially with all the dots haha. Thanks!